src/main/java/com/example/decafe/HelloController.java

```diff
@@ -0,0 +1,577 @@
+    public void changeStartCoffeeImageBack() throws FileNotFoundException {
+        startButton.setImage(createImage("startCoffee.png"));
+    }
+    // start screen - change Quit Button on mouse entered
+    public void changeQuitStartScreen() throws FileNotFoundException {
+        startQuitButton.setImage(createImage("quitEndScreenBrighter.png"));
+    }
+
+    // start screen - change Quit Button when mouse exited
+    public void changeQuitStartScreenBack() throws FileNotFoundException {
+        startQuitButton.setImage(createImage("quitEndScreen.png"));
+    }
+
+    // instructions - change GOT IT! on mouse entered
+    public void changeStartImage() throws FileNotFoundException {
+        gameStartButton.setImage(createImage("instructionsGotIt.png"));
+    }
+
+    // instructions - change GOT IT! on mouse exited
+    public void changeStartImageBack() throws FileNotFoundException {
+        gameStartButton.setImage(createImage("instructionsGotItBrighter.png"));
+    }
+
+    // end screen - change PlayAgain Button when mouse entered
+    public void changePlayAgain() throws FileNotFoundException {
+        playAgainImage.setImage(createImage("playAgainBrighter.png"));
+    }
+
+    // end screen - change PlayAgain Button when mouse exited
+    public void changePlayAgainBack() throws FileNotFoundException {
+        playAgainImage.setImage(createImage("playAgain.png"));
+    }
+
+    // end screen - change BackToStartMenu Button when mouse entered
+    public void changeBackToStartMenu() throws FileNotFoundException {
+        backToStartImage.setImage(createImage("backToStartMenuBrighter.png"));
+    }
+
+    // end screen - change BackToStartMenu Button when mouse exited
+    public void changeBackToStartMenuBack() throws FileNotFoundException {
+        backToStartImage.setImage(createImage("backToStartMenu.png"));
+    }
+
+    // end screen - change Quit Button when mouse entered
+    public void changeQuitEndScreen() throws FileNotFoundException {
+        quitEndScreenImage.setImage(createImage("quitEndScreenBrighter.png"));
+    }
+
+    // end screen - change Quit Button when mouse exited
+    public void changeQuitEndScreenBack() throws FileNotFoundException {
+        quitEndScreenImage.setImage(createImage("quitEndScreen.png"));
+    }
+
+    // if waiter is near coffee machine, change appearance when clicked
+    public void showCoffee() throws FileNotFoundException {
+        if (waiterImageView.getBoundsInParent().intersects(coffeeMachineImageView.getBoundsInParent())) {
+            Play.getCoffeeMachine().displayProduct(waiterImageView, coffeeMachineImageView, CofiBrew, progressBarCoffee);
+            File f = new File("");
+            String musicFile = f.getAbsolutePath() + File.separator + "src" + File.separator + "main" + File.separator + "resources" + File.separator + "com" + File.separator + "example" + File.separator + "decafe" + File.separator + "test_sound.wav";
```

Comment:
there is another String with the variable name musicFile at line 137, therefore renaming would avoid confusion....